membrane_curvature/base.py

```diff
@@ -110,8 +101,15 @@ class MembraneCurvature(AnalysisBase):
         gaussian_curvature = mc.results.average_gaussian_curvature
 
     The respective 2D curvature plots can be obtained using the `matplotlib`
-    package for data visualization via `imshow`. We recommend using the
-    `gaussian` interpolation.
+    package for data visualization via `contourf` or `imshow`.
+
+    For spcecific examples visit the
+    `Usage <https://membrane-curvature.readthedocs.io/en/latest/source/pages/Usage.html>`_ page.
+    Check the `Visualization
+    <https://membrane-curvature.readthedocs.io/en/latest/source/pages/Visualization.html>`_
```

Comment:
I generally use reST links instead of implicit links to keep it more readable
```reST
visit the usage_ page. Check the visualization_ page for ...

.. _usage: https://membrane-curvature.readthedocs.io/en/latest/source/pages/Usage.htm
.. _visualization: https://membrane-curvature.readthedocs.io/en/latest/source/pages/Visualization.html
```
However, given that this is the _same_ docs, you should be using sphinx refs so that the links always work:

```reST
visit the :ref:`usage-label` page. Check the :ref:`visualization-label` page for ...
```

where the docs in `Usage.rst` contain something like
```reST
.. _usage-label:

Usage
=========================================================

In this page, you can find examples of how to use MembraneCurvature to derive
curvature profiles in three types of systems:

:ref:`membrane-only`.
```

... but you're already using refs so you know what I am talking about.

Reply:
Thanks, replaced links by `:ref:` for Usage and Visualization pages.